jbpm-services/jbpm-services-ejb/jbpm-services-ejb-client/pom.xml

```diff
@@ -20,17 +20,6 @@
       <artifactId>jbpm-services-ejb-api</artifactId>
     </dependency>
     
-    <!-- uncomment to test with application deployed on JBoss EAP 6.1 -->
-    <!-- 
-    <dependency>
-      <groupId>org.jboss.as</groupId>
-      <artifactId>jboss-as-ejb-client-bom</artifactId>
-      <version>7.2.0.Final</version>
-      <optional>true</optional>
-      <type>pom</type>
-    </dependency>
-    -->
```

Comment:
I must have missed the comment, so just removed the commented code. However, we currently have separate test suite for ejb (https://github.com/droolsjbpm/jbpm/tree/master/jbpm-container-test/jbpm-remote-ejb-test), so we need to figure how many tests are being duplicated. 

Also, if we wanna keep it like this we need to upgrade to WFLY10/EAP7, we are not supporting AS7/EAP6 anymore. Also I think it may be better to add this so some specific profile (even though having too many profiles is bad as well) to avoid commented code.